tests/concepts/t3330.nim

```diff
@@ -5,29 +5,29 @@ t3330.nim(70, 4) Error: type mismatch: got <Bar[system.int]>
 but expected one of:
 proc test(foo: Foo[int])
```

Comment:
can you color expression using a different color than Error / warning / hint ?

eg for 
![image](https://user-images.githubusercontent.com/2194784/125869709-f874c256-c88e-4e55-a1e9-2c75eee84ecf.png)

Reply:
Yea this could be done, though I personally dont like the boldness in the D message as i can barely notice a difference(though it may just be your font), ideally we could have a user defined string with a `$color` as a marker for if they want colour, so then styling could be user definable using colour.